neqo-transport/src/events.rs

```diff
@@ -152,6 +158,44 @@ impl ConnectionEvents {
         self.remove(|evt| matches!(evt, ConnectionEvent::RecvStreamReadable { stream_id: x } if *x == stream_id.as_u64()));
     }
 
+    fn check_datagram_queued(&self, max_queued_datagrams: usize) {
```

Comment:
Since what this function does is not just `checking`, could you add some comments to describe this a bit?